test/components/connect.spec.tsx

```diff
@@ -2020,6 +2382,7 @@ describe('React', () => {
 
         rtl.render(
           <ProviderMock store={store}>
+            {/*// @ts-ignore */}
```

Comment:
:question: _just_ out of curiosity, what issues are we ignoring in this area of the code?

Reply:
Hmm. Yeah, it's entirely possible that's a bug in the types.

Let's leave the ignore in there for now, and maybe leave a `// TODO` comment saying to re-investigate it later?

Reply:
That's a good idea. I think I can fix it after this PR #1771  is done